util/makefile

```diff
@@ -23,7 +23,7 @@ SLANG_LIB_WIN_CL ?= $(SLANG_ROOT_WIN)/src/mw32objs/wslang32.lib
 # We assume S-Lang for Linux is installed in the correct place.
 #
 SLANG_ROOT_LINUX ?=
-SLANG_LIB_LINUX  ?= -lslang
+SLANG_LIB_LINUX  ?= $(SLANG_ROOT_LINUX)/src/objs/libslang.a
```

Comment:
You have done this on Linux? If so you have an env-var `SLANG_ROOT_LINUX` set? 
Maybe there should be a comment for that.